src/core/runtime/config-nodejs.js

```diff
@@ -20,6 +20,7 @@ module.exports = () => ({
     }
   },
   Bootstrap: [
+    '/ip4/104.131.131.82/tcp/4001/p2p/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ',
```

Comment:
@mburns if js-ipfs supports using the `p2p` protocol name rather than `ipfs` then it'd be best to use `p2p` consistently for all the entries here.

Reply:
As of 0.41.0 it does!